Formula/hyperkit.rb

```diff
@@ -0,0 +1,56 @@
+class Hyperkit < Formula
+  desc "Toolkit for embedding hypervisor capabilities in your application"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit.git",
+    :tag => "v0.20180123",
+    :revision => "8bbe1da0166553a5b0ccff170bf9ccbd7d10753e"
+
+  head do
+    url "https://github.com/moby/hyperkit.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  option "with-qcow", "Enable support for qcow disk image format"
+  if build.with? "qcow"
+    depends_on "ocaml" => :build
+    depends_on "opam" => :build
+    depends_on "libev"
+  end
+
+  depends_on :xcode => ["9.0", :build]
+  depends_on :macos => :yosemite
+
+  def install
+    if build.with? "qcow"
+      system "opam", "init", "--no-setup"
+      opam_dir = "#{buildpath}/.brew_home/.opam"
+      ENV["CAML_LD_LIBRARY_PATH"] = "#{opam_dir}/system/lib/stublibs:/usr/local/lib/ocaml/stublibs"
+      ENV["OPAMUTF8MSGS"] = "1"
```

Comment:
Do we need that one?

Reply:
those values come from `opam config env`, they are necessary for `opam` to work properly.